imcsdk/apis/server/power.py

```diff
@@ -18,6 +18,30 @@
 from imcsdk.imccoreutils import get_server_dn
 from imcsdk.imcexception import ImcOperationError
 
+import logging
+log = logging.getLogger('imc')
+
+# This list is currently maintained manually.
+# Ideally all such config/capabilites should come from a capability file
+supported_models = ['UCSC-C220-M4', 'UCSC-C240-M4', 'UCSC-C3X60-M4']
+
+
+def is_supported_model(handle):
+    for model in supported_models:
+        if handle.model.find(model) != -1:
+            return True
+    log.info("Current Model:'%s' does not support Power Cap and Budgeting" %
+             handle.model)
+    return False
+
+
+def get_supported_models(handle):
```

Comment:
API name is misleading here - `get_supported_models` would mean `return supported_models`